### PR TITLE
repo: Update our default Pull Request template

### PR DESCRIPTION
**Pull request by sandrade-dcl**
## What this PR includes?

An update of our default Pull Request template for the `unity-renderer` repo.

## How to test it?

N/A

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
+## What this PR includes?
 <!-- 
-Please refer to the issue or JIRA ticket to close like `Closes #14` or 
-`Fixes [CLIENT-1]`
+Explain what this PR implements:
+In case you are fixing any specific issue, please refer to it with `Fixes #issue_number`.
+In case you are implementing a new feature, please write a brief description about it.
+Optionally, you can add any useful external reference that you think could help to documentate a bit more the implementation (for example: design/architecture documents, figma links, screenshots, etc.)
```
Comment (BrianAmadori) [inline .github/PULL_REQUEST_TEMPLATE.md:6]:
```suggestion
Optionally, you can add any useful external reference that you think could help to documentate a bit more the implementation (for example: design/architecture documents, figma links, screenshots, etc.)
```
This sentence phrasing is a bit weird and `documentate` should be `document`.

Maybe it's easier to read if we rephrase it to:
```
As an optional step, you can link or add any useful external documentation to give more context about the proposed changes. (for example: design/architecture documents, figma links, screenshots, etc.)
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-
-# What? <!-- what is this PR? -->
 ...
 
-# Why? <!-- Explain the reason -->
-...
+## How to test it?
+<!-- 
+Explain how to test the feature (or fix) for someone who doesn't know anything about this implementation:
+At very least add the specific url from which to test the build and add to it any param you think it wuld be needed.
```
[the remark this answers is not in the document]
Comment (sandrade-dcl) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done in https://github.com/decentraland/unity-renderer/pull/355/commits/b886306674708ad607224f5ac2f1906aa9b07c97, thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sandrade-dcl) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done in https://github.com/decentraland/unity-renderer/pull/355/commits/b886306674708ad607224f5ac2f1906aa9b07c97, thanks!

**Review by Suduck — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
-
-# What? <!-- what is this PR? -->
 ...
 
-# Why? <!-- Explain the reason -->
-...
+## How to test it?
+<!-- 
+Explain how to test the feature (or fix) for someone who doesn't know anything about this implementation:
+At very least add the specific URL from which to test the build and add to it any param you think it would be needed.
```
Comment (AjimenezDCL) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: This should be done by the CI automatically, anyway I agree that if any param is needed, that automatic comment with the URL should be edited to reflect it.

**Review by AjimenezDCL — APPROVED** [review]

**Review by BrianAmadori — APPROVED** [review]
